pallets/cash/src/lib.rs

```diff
@@ -191,6 +193,12 @@ decl_storage! {
 
         /// Validator spread due to miner of last block.
         LastMinerSharePrincipal get(fn last_miner_share_principal): CashPrincipalAmount;
+
+        /// The timestamp of the previous block or defaults to timestamp at genesis.
+        LastBlockTimestamp get(fn last_block_timestamp): Timestamp;
```

Comment:
dev note - not sure if we want `config()` here and/or how disruptive that is for the time being

Reply:
Yea good call out, this is the question